numba_dppy/dppy_parfor_diagnostics.py

```diff
@@ -0,0 +1,104 @@
+from numba.parfors.parfor import ParforDiagnostics, _termwidth, print_wrapped
+
+
+class ExtendedParforDiagnostics(ParforDiagnostics):
+    def __init__(self):
+        ParforDiagnostics.__init__(self)
+        self.extra_info = {}
+
+    def dump(self, level=1):
+        ParforDiagnostics.dump(self, level=1)
```

Comment:
Why are we hardcoding level to 1 here?

Reply:
In numba's dump function there is should be defined level
If we don't do it, level=0 and "Report level unknown, should be one of 1, 2, 3, 4"

Reply:
level is set to 1 by default in line 9, can we not pass that to ParforDiagnostics?

Reply:
When we specify the level value in the environment variable, then it is transmitted, and by default (if nothing is specified) level 1 works

Reply:
@1e-to I think comment is about line 10
```python
ParforDiagnostics.dump(self, level=1)
```
Shouldn't it be:
```python
ParforDiagnostics.dump(self, level=level)
```
?

Reply:
Fixed

Reply:
Also isn't it:

```python
super().dump(level)
```

?